recipes/libgif/libgif.inc

```diff
@@ -0,0 +1,26 @@
+DESCRIPTION = "GIFLIB is a package of portable tools and library routines for working with GIF images."
+HOMEPAGE = "http://giflib.sourceforge.net/"
+LICENSE = "MIT"
+
+SRC_URI = "http://kent.dl.sourceforge.net/project/giflib/giflib-${PV}.tar.bz2"
+
+inherit autotools library auto-package-utils
+
+S = "${SRCDIR}/giflib-${PV}"
```

Comment:
oh, do not name the recipe libgiff when upstream calls the software giflib, that just makes it harder to find stuff